website/source/style/components/_markdown.scss

```diff
@@ -0,0 +1,163 @@
+$lsg--color: $black;
+$lsg--background-color: $white;
+$lsg--code-background-color: mix(white, $turquoise, 90%);
+$lsg--code-color: $pink;
+$lsg--code-font: courier new, courier, monospace;
+$lsg--base-font-size: 13px;
+$lsg--base-font-weight: normal;
+$lsg--base-line-height: 1.1;
+$lsg--base-text-align: left;
+$lsg--code-font-size: 13px;
+$lsg--code-border-radius: 0;
+$lsg--code-line-height: 1.1;
+$lsg--code-font-weight: normal;
+$lsg--width: 600px;
+$lsg--gap-width: 20px;
+$lsg--highlight-color: $yellow;
+$lsg--highlight-border-radius: 0;
+$lsg--base-font: gubia, helvetica neue, helvetica, arial;
+$lsg--headline-font: gubia, helvetica neue, helvetica, arial;
+$lsg--headline-font-size: 36px;
+$lsg--headline-font-weight: bold;
+$lsg--headline-line-height: 1.3;
+$lsg--headline-text-align: center;
+$lsg--page-title-font-size: 44px;
+$lsg--sub-headline-font-size: 22px;
+$lsg--sub-sub-headline-font-size: 16px;
+$lsg--border-radius: 0;
+$lsg--layout-selector: ".layout";
+$lsg--full-width-padding: 0;
+$lsg--button-width: 23px;
+$lsg--button-height: 23px;
+$lsg--copy-background-image: none;
+
+@import "livingstyleguide/reset";
+@import "livingstyleguide/content";
+@import "livingstyleguide/code";
+
+.markdown {
+  max-width: $lsg--width + 2 * 40px;
+  padding: 40px;
+  overflow: hidden;
+  text-align: left;
+
+  a {
+    color: darken($turquoise, 20%);
+    display: inline;
+
+    &:hover {
+      color: darken($turquoise, 30%);
+    }
+  }
+
+  strong {
+    @extend %display-inline;
+    @extend %font-weight-bold;
+  }
+
+  em {
+    @extend %display-inline;
+    @extend %font-weight-bold;
+  }
+
+  .lsg--headline {
+    font-size: 3em;
+    margin: 0 0 0.5em;
+    width: auto;
+  }
+
+  .lsg--sub-headline {
+    font-size: 2em;
+    margin: 1.5em 0 0.5em;
+    width: auto;
+  }
+
+  p,
+  .lsg--paragraph,
+  .lsg--ordered-list,
+  .lsg--ordered-list-item,
+  .lsg--unordered-list,
+  .lsg--unordered-list-item {
+    font-size: 18px;
+    font-family: source sans pro, arial, helvetica, sans-serif;
+    font-weight: 200;
+    line-height: 1.5;
+    margin: 0 0 0.75em;
+    width: auto;
+  }
+
+  .lsg--unordered-list {
+    counter-reset: ul;
+    margin: 0 0 0.75em;
+  }
+
+  .lsg--ordered-list-item,
+  .lsg--unordered-list-item {
+    list-style: none;
+    margin-left: 3em;
+  }
+
+  .lsg--unordered-list-item {
+    &:before {
+      background: $turquoise;
```

Comment:
Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets